content/en/docs/helm/helm_template.md

```diff
@@ -32,6 +32,7 @@ helm template [NAME] [CHART] [flags]
       --dry-run                    simulate an install
   -g, --generate-name              generate the name (and omit the NAME parameter)
   -h, --help                       help for template
+      --include-crds               output CRDs along with templated manifests 
```

Comment:
output CRDs along with templated manifests --> include CRDs in the templated output